### PR TITLE
Fish autocompletion installation instructions

### PR DESCRIPTION
**Pull request by kevgo**
@charlierudolph @allewun 

This displays instructions on how to install Fish shell autocompletion after the installation.

Depends on https://github.com/Originate/git-town/pull/358

Implements https://github.com/Originate/git-town/issues/284

### CODE REVIEW

**Comment by kevgo** [comment]
@charlierudolph @allewun This PR is unblocked now. Please review!

File: git-town.rb [hunk 1]
```diff
@@ -19,4 +19,23 @@ def install
...
+      run "git town install-fish-autocompletion"
+      in the terminal.
+
+      To install the completions manually, make
+      #{libexec}/autocomplete/git.fish
+      available as ~/.config/fish/completions/git.fish.
+
+      In a standard setup, this looks like:
+      mkdir -p ~/.config/fish/completions
+      ln -s #{libexec}/autocomplete/git.fish ~/.config/fish/completions/git.fish
```
Comment (allewun) [inline git-town.rb]: Probably should've caught this in your earlier PR, but 2 questions:

1. What if the user already has git.fish?
2. Sine the autocompletion file is already on disk, why not just use `cp` [here](https://github.com/Originate/git-town/blob/351f0bec4469974ed193c133ac652ddf75c42a70/src/git-town#L8)?

Comment (kevgo) [inline git-town.rb]: Re 1: Fish comes with Git autocompletion out of the box, so if the user already has this file, its most likely an older version of our autocompletion, and should be overwritten. Let's wait for bug reports before we make this more complicated.

Re 2: A symlink also points to the file on disk, but has the advantage that when we update Git Town, the linked autocompletion automatically gets updated as well.

Comment (charlierudolph) [inline git-town.rb]: Can we name the file `git-town.fish` just to make it clear that the file was added by git-town and not git?

Comment (allewun) [inline git-town.rb]: Re 2: I was referring to the link (click here)

Comment (kevgo) [inline git-town.rb]: @charlierudolph unfortunately that's not possible. The file name must match the command being autocompleted.

Comment (kevgo) [inline git-town.rb]: @allewun okay thanks for pushing me to do this. I have submitted #406 that uses the locally installed file, but symlinks to it, and doesn't override existing files here.

**Comment by kevgo** [comment]
all comments addressed

**Pull request merged**